assets/blocks/quiz/answer-blocks/index.js

```diff
@@ -0,0 +1,56 @@
+import { __ } from '@wordpress/i18n';
+import { applyFilters } from '@wordpress/hooks';
+
+/**
+ * @typedef QuestionType
+ *
+ * @property {string}   title       Question type name.
+ * @property {string}   description Question type description.
+ * @property {Function} edit        Editor component.
+ */
+
+/**
+ * Question type definitions.
+ *
+ * @type {Object.<string, QuestionType>}
+ */
+const questionTypes = {
```

Comment:
Should we capitilize this as it is a const?

Reply:
This isn't really a constant, just an ordinary variable.